scio-parquet/src/main/scala/com/spotify/scio/parquet/read/ParquetRead.scala

```diff
@@ -43,8 +58,92 @@ object ParquetRead {
     readSupportFactory: ReadSupportFactory[T],
     conf: SerializableConfiguration,
     projectionFn: SerializableFunction[T, R]
-  ): ParDo.SingleOutput[ReadableFile, R] = {
+  ): PTransform[PCollection[ReadableFile], PCollection[R]] = {
```

Comment:
I think this will technically break MiMA :( The reason is that `ParDo.of(..)` constructs a `PTrasnform[PCollection[_ <: ReadableFile], ...]`. Thus, the pre-existing signature didn't work when used with SCollection#readFiles as the signatures didn't match.

Reply:
maybe candidate to 0.13?